gumby/modules/experiment_module.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+
+class ExperimentModule(object):
+    def __init__(self, experiment):
+        super(ExperimentModule, self).__init__()
+        self._logger = logging.getLogger(self.__class__.__name__)
+        self.experiment = experiment
+        experiment.experiment_modules.append(self)
+        self._logger.info("Load experiment module %s", self.__class__.__name__)
+
+    @classmethod
+    def on_module_load(cls, experiment):
+        pass
+
+    def print_on_change(self, name, prev_dict, cur_dict):
+        return self.experiment.print_on_change(name, prev_dict, cur_dict)
+
+    @property
+    def my_id(self):
+        return self.experiment.my_id
+
+    @property
+    def vars(self):
+        return self.experiment.vars
+
+    @property
+    def all_vars(self):
+        return self.experiment.all_vars
+
+    def on_id_received(self):
+        pass
+
+    @staticmethod
+    def str2bool(v):
+        return v.lower() in ("yes", "true", "t", "1")
+
+    @staticmethod
+    def str2tuple(v):
```

Comment:
Yes, it is used to parse True/False style arguments to the scenario callback functions. For example `should_become_exit` in the hidden services module uses it.